### ISSUE TITLE
Compression is broken

### ISSUE THREAD

**Issue by takase1121** [zhanghai/MaterialFiles#317]
I try to use the "Compress" option, but it gave me an error.

Logcat: [files.log](https://github.com/zhanghai/MaterialFiles/files/4841556/files.log)

**Comment by zhanghai** [comment]
```
06-28 10:07:52.224 13771 13927 W System.err: java8.nio.file.NoSuchFileException: /Wallpapers/temp/temp.zip: java.io.FileNotFoundException: Cannot find document ID for /Wallpapers/temp/temp.zip
06-28 10:07:52.226 13771 13927 W System.err: 	at f.a.a.a.o.e.b.b(:24)
06-28 10:07:52.226 13771 13927 W System.err: 	at q.a.c.l.b(:11)
06-28 10:07:52.226 13771 13927 W System.err: 	at j.a.s1.b(Unknown Source:14)
06-28 10:07:52.226 13771 13927 W System.err: 	at j.a.s1.q(:2)
06-28 10:07:52.226 13771 13927 W System.err: 	at f.a.a.a.j.s$b.run(:2)
06-28 10:07:52.226 13771 13927 W System.err: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
06-28 10:07:52.226 13771 13927 W System.err: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
06-28 10:07:52.226 13771 13927 W System.err: 	at java.lang.Thread.run(Thread.java:764)
06-28 10:07:52.227 13771 13927 W System.err: Caused by: me.zhanghai.android.files.provider.content.resolver.ResolverException: java.io.FileNotFoundException: Cannot find document ID for /Wallpapers/temp/temp.zip
06-28 10:07:52.227 13771 13927 W System.err: 	at f.a.a.a.o.e.d.a.f(:1)
06-28 10:07:52.227 13771 13927 W System.err: 	at f.a.a.a.o.e.d.a.c(Unknown Source:2)
06-28 10:07:52.227 13771 13927 W System.err: 	at f.a.a.a.o.e.b.b(:20)
06-28 10:07:52.227 13771 13927 W System.err: 	... 7 more
06-28 10:07:52.227 13771 13927 W System.err: Caused by: java.io.FileNotFoundException: Cannot find document ID for /Wallpapers/temp/temp.zip
06-28 10:07:52.227 13771 13927 W System.err: 	... 10 more
06-28 10:07:53.027 13771 14076 W System.err: java.nio.channels.NonWritableChannelException
06-28 10:07:53.033 13771 14076 W System.err: 	at sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:236)
06-28 10:07:53.033 13771 14076 W System.err: 	at q.a.a.b.write(Unknown Source:2)
06-28 10:07:53.033 13771 14076 W System.err: 	at java.nio.channels.Channels.writeFullyImpl(Channels.java:79)
06-28 10:07:53.033 13771 14076 W System.err: 	at java.nio.channels.Channels.writeFully(Channels.java:102)
06-28 10:07:53.033 13771 14076 W System.err: 	at java.nio.channels.Channels.access$000(Channels.java:62)
06-28 10:07:53.033 13771 14076 W System.err: 	at java.nio.channels.Channels$1.write(Channels.java:175)
06-28 10:07:53.033 13771 14076 W System.err: 	at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)
06-28 10:07:53.033 13771 14076 W System.err: 	at java.io.BufferedOutputStream.write(BufferedOutputStream.java:126)
06-28 10:07:53.033 13771 14076 W System.err: 	at u.a.a.a.a.o.o.a(:2)
06-28 10:07:53.033 13771 14076 W System.err: 	at u.a.a.a.a.o.o.a(:1)
06-28 10:07:53.033 13771 14076 W System.err: 	at u.a.a.a.a.o.g0.write(:6)
06-28 10:07:53.033 13771 14076 W System.err: 	at j.a.s1.a(Unknown Source:42)
06-28 10:07:53.033 13771 14076 W System.err: 	at f.a.a.a.o.b.j.b.a(:21)
06-28 10:07:53.033 13771 14076 W System.err: 	at j.a.s1.a(:1)
06-28 10:07:53.033 13771 14076 W System.err: 	at f.a.a.a.i.c.b(:3)
06-28 10:07:53.033 13771 14076 W System.err: 	at q.a.c.l.a(:30)
06-28 10:07:53.033 13771 14076 W System.err: 	at q.a.c.l.a(Unknown Source:9)
06-28 10:07:53.033 13771 14076 W System.err: 	at f.a.a.a.i.d.b(:4)
06-28 10:07:53.033 13771 14076 W System.err: 	at me.zhanghai.android.files.filejob.FileJobService$b.run(:1)
06-28 10:07:53.033 13771 14076 W System.err: 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:458)
06-28 10:07:53.034 13771 14076 W System.err: 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
06-28 10:07:53.034 13771 14076 W System.err: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
06-28 10:07:53.034 13771 14076 W System.err: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
06-28 10:07:53.034 13771 14076 W System.err: 	at java.lang.Thread.run(Thread.java:764)
06-28 10:07:53.034 13771 14076 W System.err: 	Suppressed: java.io.IOException: This archive contains unclosed entries.
06-28 10:07:53.034 13771 14076 W System.err: 		at u.a.a.a.a.o.g0.d(:115)
06-28 10:07:53.034 13771 14076 W System.err: 		at f.a.a.a.o.b.j.b.close(Unknown Source:2)
06-28 10:07:53.034 13771 14076 W System.err: 		at l.b.k.m.a(Unknown Source:9)
06-28 10:07:53.034 13771 14076 W System.err: 		at f.a.a.a.i.d.b(:5)
06-28 10:07:53.034 13771 14076 W System.err: 		... 6 more
```

**Comment by zhanghai** [comment]
Thanks for reporting this bug! Fixed in 90b26fd.
